x-pack/plugins/apm/server/lib/helpers/setup_request.ts

```diff
@@ -45,23 +61,24 @@ export function setupRequest(req: Legacy.Request): Setup {
   const client: ESClient = async (type, params) => {
     const includeFrozen = await uiSettings.get('search:includeFrozen');
 
+    const nextParams = {
+      ...addFilterForLegacyData(params), // filter out pre-7.0 data
+      ignore_throttled: !includeFrozen, // whether to query frozen indices or not
+      rest_total_hits_as_int: true // ensure that ES returns accurate hits.total with pre-6.6 format
+    };
+
     if (query._debug) {
       console.log(`DEBUG ES QUERY:`);
-      console.log({ includeFrozen });
+      console.log('includeFrozen: ', includeFrozen);
```

Comment:
Am I missing something here or just personal preference?

Reply:
Yes, changed it for no good reason. Can change it back.